### PR TITLE
Test example.com, googleapis.com, and content-autofill.googleapis.com

### PR DESCRIPTION
**Pull request by justincr-elastic**
Add extra tests to RegisteredDomainProcessorTests.java.

See #67611 when tests were originally added.

### CODE REVIEW

File: modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/RegisteredDomainProcessorTests.java [hunk 1]
```diff
@@ -41,6 +41,15 @@ public void testBasic() throws Exception {
             "1"
         );
         testRegisteredDomainProcessor(buildEvent("www.books.amazon.co.uk"), "www.books.amazon.co.uk", "amazon.co.uk", "co.uk", "www.books");
+        testRegisteredDomainProcessor(buildEvent("example.com"), "example.com", "example.com", "com", null);
+        testRegisteredDomainProcessor(buildEvent("googleapis.com"), "googleapis.com", null, "googleapis.com", null);
```
Comment (danhermann) [inline modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/RegisteredDomainProcessorTests.java:45]: Could you move the above `fastly.net` test here and then add a comment to the effect that it, the `googleapis.com` test, and the `content-autofill...` test are testing third-party eTLDs? I think that will clarify the purpose of those tests and illustrate why they behave differently than the other tests.

**Review by danhermann — APPROVED** [review]
I added one comment below about re-ordering one of the tests and adding a comment about their intent. If you can do that, this PR looks good to me.

File: modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/RegisteredDomainProcessorTests.java (hunk 1 above)
Comment (justincr-elastic) [reply modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/RegisteredDomainProcessorTests.java:48]: Resolved.

**Comment by justincr-elastic** [comment]
@elasticmachine update branch

**Comment by justincr-elastic** [comment]
@elasticmachine update branch

**Comment by danhermann** [comment]
LGTM 👍

**Comment by justincr-elastic** [comment]
@elasticmachine update branch

**Pull request merged**